### PR TITLE
Fix replace IPv4 route decrypt rule failed

### PR DESCRIPTION
**Pull request by aanm**
Probably not the right fix for https://github.com/cilium/cilium/issues/7041 but I'll leave to @jrfastab to decide

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7046)

### CODE REVIEW

File: Documentation/gettingstarted/encryption.rst [hunk 1]
```diff
@@ -34,29 +34,21 @@ keys may be manually distributed but that is not shown here.
...
 AES-CBC with HMAC-256 (hash based authentication code), but any of the supported
 Linux algorithms may be used. To generate use the following
 
 .. parsed-literal::
-  KEY1=0x`dd if=/dev/urandom count=16 bs=1 2> /dev/null| xxd -p -c 64`
-  KEY2=0x`dd if=/dev/urandom count=16 bs=1 2> /dev/null| xxd -p -c 64`
-  echo "  keys: \\"hmac(sha256) $KEY1 cbc(aes) $KEY2\\"" >> cilium-ipsec-keys.yaml
+
+    kubectl create -n kube-system secret generic cilium-ipsec-keys \\
+       --from-literal=keys="hmac(sha256) $(echo `dd if=/dev/urandom count=32 bs=1 2> /dev/null| xxd -p -c 64`) cbc(aes) $(echo `dd if=/dev/urandom count=32 bs=1 2> /dev/null| xxd -p -c 64`)"
```
Comment (jrfastab) [inline Documentation/gettingstarted/encryption.rst]: count=32 -> count=16

**Comment by aanm** [comment]
test-me-please

**Review by jrfastab — APPROVED** [review]
thanks LGTM. I have the same patch on my stack.

**Comment by aanm** [comment]
test-me-please

**Pull request merged**